src/ebay-star-rating-select/__tests__/index.stories.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import { EbayStarRatingSelect } from '../'
+import { IsolatedDefaultProps, FieldsetDefaultProps } from './mocks'
+
+
+export default {
+    component: EbayStarRatingSelect,
+    title: 'ebay-star-rating-select',
+    argTypes: {
+        disabled: {
+            control: { type: 'boolean' },
+        },
+        value: {
+            control: { type: 'number' },
+            description:
+                '1 - 5, depending on how many stars are selected. If 0 or null defaults to no stars selected',
+        },
+        a11yStarText: {
+            control: 'object',
+            description: 'Array object which sets the aria label for each star',
+        },
```

Comment:
There is a problem on the storybook, it failed to show the value `a11yStarText`.

For example, the Marko storybook, the value is visible on storybook knob:

![Screenshot 2022-08-23 at 15 43 00](https://user-images.githubusercontent.com/1072047/186174058-388ac8f5-f0fa-4ceb-af3b-829a97bcd848.png)

Meanwhile, our storybook failed to show the value:

![Screenshot 2022-08-23 at 15 43 17](https://user-images.githubusercontent.com/1072047/186174227-c15445d7-9573-4f15-8c09-e1e42f9498d8.png)

This might be problem since user might confuse what value should we put for `a11yStarText` (example: `['1 star', '2 stars', '3 stars', '4 stars', '5 stars']`). I would put this example value on the readme as well.